nixos/tests/gitea.nix

```
@@ -18,11 +18,11 @@ with pkgs.lib;
       };
 
     testScript = ''
-      startAll;
+      start_all
```

```suggestion
      start_all()
```

otherwise it's not a function call but a NOP